src/Model/GamePackage/Game.java

```diff
@@ -310,52 +307,69 @@ public void movePlayer(Player player, int dieVal1, int dieVal2) {
      * One player sells one of their ownableSquares to another player
      * @param ownableSquare The ownable square being sold
      * @param buyer The player buying the ownable square
+     * @return whether the action was able to be completed )
      *********************************************************************/
-    public void sellOwnableSquare(OwnableSquare ownableSquare, Player buyer) {
+    public boolean sellOwnableSquare(OwnableSquare ownableSquare, Player buyer) {
         // Check if buyer paid full amount
         if( buyer.pay(ownableSquare.getPRICE()) == ownableSquare.getPRICE()){
             // Buyer receives ownableSquare, Owner gives it.
             buyer.recieveProperty(ownableSquare.getOwner().giveProperty(ownableSquare));
         }
         else{
-            // TODO: Call a function that handles when a buyer only has partial (if any) of the money during a trade
+            return false;
         }
+        return true;
     }
 
-    public void mortgageProperty(PropertySquare property) {
-        //TODO: finish body
-        //Display the list of properties the player has.
+    /**********************************************************************
+     * Sets the given property to mortgaged and pays the owner the respective
+     * amount.
+     *
+     * @param property the property that is about to be mortgaged
+     *********************************************************************/
+    public void mortgageProperty(OwnableSquare property) {
+       property.setMortgaged(true);
+       property.getOwner().receiveMoney(property.getMORTGAGE_VAL());
     }
 
-    public void payMortgage(){
-        //TODO: finish body
-    }
+    /**********************************************************************
+     * This methods allows a player to the property mortgage and set the
+     * mortgage property to false and discount the money from the player's
+     * wallet.
+     *
+     * @param propertySquare the property that the mortgage is being lifted.
+     *********************************************************************/
+    public void payMortgage(PropertySquare propertySquare){
+       propertySquare.setMortgaged(false);
+       propertySquare.getOwner().pay((int)Math.ceil(propertySquare.getMORTGAGE_VAL() * 1.1));
+     }
 
-    public void auctionProperty() {
```

Comment:
If this functionality was removed here, is it implemented somewhere else?

Reply:
We decided this was going to be implemented by the GUI/controller. This method will just make calls to many other Game methods based on use input. Same with Trade.